cws-service/src/main/java/jpl/cws/console/WorkerMonitorBackgroundThread.java

```diff
@@ -17,10 +19,14 @@
 	
 	@Autowired private SchedulerDbService schedulerDbService;
 	@Autowired private ExternalTaskService externalTaskService;
-	
+
+	@Value("${cws.num.days.after.to.remove.abandoned.workers}")		private String numDaysAfterRemoveDeadWorkers;
+	@Value("${cws.history.days.to.live}") 	private String historyDaysToLive;
+
 	private static final int THRESHOLD_MILLIS_FOR_DEAD_WORKER = 60000;
 	private static final int THIRTY_SECONDS = 30000;
```

Comment:
```suggestion
	private static final int THIRTY_SECONDS = 30000;
	private static final int ONE_DAY = 1000 * 60 * 60 * 24;
```

instead of just putting `86400000` as a magic number